readthedocs/urls.py

```diff
@@ -26,7 +27,7 @@
         TemplateView
         .as_view(template_name='security.txt', content_type='text/plain'),
     ),
-    url(r'^support/$', SupportView.as_view(), name='support'),
+    url(r'^support/$', login_required(SupportView.as_view()), name='support'),
```

Comment:
We use this mxin in our code base https://github.com/readthedocs/readthedocs.org/blob/3c9dafe093ff444fa304f660b661ad0be3e6316f/readthedocs/core/mixins.py#L20-L20

Reply:
Gotcha, should probably chagne https://github.com/readthedocs/readthedocs.org/blob/1758dc9234ce18e0fcb4076fd37a400724e99951/readthedocs/projects/urls/private.py#L66-L65 as well.

Reply:
Looks like Django has this builtin now: https://docs.djangoproject.com/en/dev/topics/auth/default/#django.contrib.auth.mixins.LoginRequiredMixin

Reply:
yeah, our mixin basically inherits from that, I think we can use django's mixin directly, don't think we would be adding any other logic there.